Library/Homebrew/cask/lib/hbc/installer.rb

```diff
@@ -225,7 +225,12 @@ def print_caveats
   def enable_accessibility_access
     return unless @cask.accessibility_access
     ohai "Enabling accessibility access"
-    if MacOS.version <= :mountain_lion
+    if MacOS.version >= :sierra
+      opoo <<-EOS.undent
+        Access denied to accessibility DB (TCC.db) by SIP on this version of macOS.
+        If needed, enable manually via the app or System Preferences.
+      EOS
+    elsif MacOS.version <= :mountain_lion
```

Comment:
I'd change the wording to:

Accessibility access cannot be enabled on this version of macOS.
Enable it manually via System Preferences.

Same with “disable”.